go/worker/helpers.go

```diff
@@ -379,7 +379,9 @@ func stoppedDirtyToStoppedClean(olPath string) error {
 		// Log an error if unmounting the main directory fails.
 		if errors.Is(err, syscall.EINVAL) {
 			fmt.Printf("Sandbox mount root is not mounted. No need to clean up.\n")
-		} else {
+		} else if errors.Is(err, syscall.ENOENT) { // added this case
```

Comment:
A comment like "added this case" makes sense for somebody reading the PR, but we want to write a comment for somebody who is browsing around the code in the future.  Then it wouldn't make sense, because everything was added at some point, of course.